target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -381,6 +381,11 @@ u7628-01-128M-16M)
 	set_usb_led "u7628-01:green:usb"
 	set_wifi_led "u7628-01:green:wlan"
 	;;
+var11n-300)
+	ucidef_set_led_default "power" "power" "$board:blue:status" "1"
```

Comment:
This LED doesn't exist. There is a `:blue:system` instead. Nevertheless, please add the status/system LED to diag.sh.